pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -33,16 +33,16 @@ const (
 type CommonPrometheusFields struct {
 	// PodMetadata configures Labels and Annotations which are propagated to the prometheus pods.
 	PodMetadata *EmbeddedObjectMetadata `json:"podMetadata,omitempty"`
-	// ServiceMonitors to be selected for target discovery. *Deprecated:* if
-	// neither this nor podMonitorSelector are specified, configuration is
-	// unmanaged.
+	// ServiceMonitors to be selected for target discovery. *Note:* if
+	// neither this nor podMonitorSelector are specified, prometheus-operator
+	// will not manage the configuration.
 	ServiceMonitorSelector *metav1.LabelSelector `json:"serviceMonitorSelector,omitempty"`
 	// Namespace's labels to match for ServiceMonitor discovery. If nil, only
 	// check own namespace.
 	ServiceMonitorNamespaceSelector *metav1.LabelSelector `json:"serviceMonitorNamespaceSelector,omitempty"`
 	// *Experimental* PodMonitors to be selected for target discovery.
-	// *Deprecated:* if neither this nor serviceMonitorSelector are specified,
-	// configuration is unmanaged.
+	// *Note:* if neither this nor serviceMonitorSelector are specified,
```

Comment:
nit: should we also add this for probes, to make it consistent?

Reply:
you're right! I had missed probes in the condition, will update, thanks 👍